projects/observability/src/shared/components/interval-select/interval-select.component.ts

```diff
@@ -23,7 +23,7 @@ import { SelectOption, SelectSize } from '@hypertrace/components';
 })
 export class IntervalSelectComponent implements OnChanges {
   @Input()
-  public interval?: IntervalValue;
+  public interval: IntervalValue = 'AUTO';
```

Comment:
Why change this default? It's a breaking change and I'm not sure what it adds since the widget is providing a default if it wants to.

Reply:
Yeah, I can revert it.

Reply:
done